<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quản lý dịch vụ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	background-color:rgb(255, 255, 127);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:rgb(0, 85, 255);
	border-radius: 15px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Quản lý dịch vụ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>431</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
	background-color:rgb(85, 255, 127);
	font-size:18px;
	border-radius:10px;
}</string>
     </property>
     <property name="title">
      <string>Thông tin dịch vụ</string>
     </property>
     <widget class="QLineEdit" name="txtIDichVu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>ID dịch vụ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtITenDichVu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Tên dịch vụ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtGiadichvu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Giá dịch vụ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPhidichvu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Phí dịch vụ...</string>
      </property>
     </widget>
    </widget>
    <widget class="QListView" name="lstDichvu">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>531</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThemd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>460</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSuad">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>460</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 170, 0);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color:rgb(0, 255, 127);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnXoad">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>560</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(255, 0, 0);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(255, 84, 84);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThoatd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>560</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size: 20px;
    border-radius: 10px;
	background-color: rgb(0, 85, 255);
	color:rgb(255, 255, 255);
	font-weight: bold;
}
QPushButton:hover{
	background-color: rgb(0, 170, 255);
	cursor: pointer;
}</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
